packages/terra-clinical-label-value-view/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 ## Unreleased
+* Added
+  * Added a new prop `topicTextId` on the LabelValueView component.
```

Comment:
This should be updated with the new prop name.

Reply:
Right! Made the change. Thanks!